static/src/javascripts/projects/common/modules/commercial/creatives/fluid250.js

```diff
@@ -22,6 +24,34 @@ define([
         this.params = params;
     };
 
+    /**
+     * TODO: rather blunt instrument this, due to the fact *most* mobile devices don't have a fixed
+     * background-attachment - need to make this more granular
+     */
+    Fluid250.hasScrollEnabled = !detect.isIOS() && !detect.isAndroid();
+
+    Fluid250.prototype.updateBgPosition = function () {
+
+        var bgpositionx, layeronebgpositionx, layertwobgpositionx, layerthreebgpositionx;
+
+        if (this.params.backgroundPosition !== '') {
+            bgpositionx = this.params.backgroundPosition.split(' ')[0];
+            $('.ad-slot--top-banner-ad .creative--fluid250').css('background-position', bgpositionx + ' ' + window.pageYOffset * (this.params.backgroundImageScroll / 100) + 'px').css('background-repeat', 'repeat-y');
+        }
+        if (this.params.layerOneBGPosition !== '') {
+            layeronebgpositionx = this.params.layerOneBGPosition.split(' ')[0];
+            $('.ad-slot--top-banner-ad .creative--fluid250 .fluid250_layer1').css('background-position', layeronebgpositionx + ' ' + window.pageYOffset * (this.params.layerOneImageScroll / 100) + 'px').css('background-repeat', 'repeat-y');
+        }
+        if (this.params.layerTwoBGPosition !== '') {
+            layertwobgpositionx = this.params.layerTwoBGPosition.split(' ')[0];
+            $('.ad-slot--top-banner-ad .creative--fluid250 .fluid250_layer2').css('background-position', layertwobgpositionx + ' ' + window.pageYOffset * (this.params.layerTwoImageScroll / 100) + 'px').css('background-repeat', 'repeat-y');
+        }
+        if (this.params.layerThreeBGPosition !== '') {
+            layerthreebgpositionx = this.params.layerThreeBGPosition.split(' ')[0];
+            $('.ad-slot--top-banner-ad .creative--fluid250 .fluid250_layer3').css('background-position', layerthreebgpositionx + ' ' + window.pageYOffset * (this.params.layerThreeImageScroll / 100) + 'px').css('background-repeat', 'repeat-y');
+        }
+    };
+
```

Comment:
Would be nice to see some FastDOM love here, i.e. I can see lots of reading an writing to the DOM.